src/components/activities/RosterPanel.tsx

```diff
@@ -55,11 +58,42 @@ export function RosterRowCard({ status, children, onClick, ...props }: PaperProp
   );
 }
 
+function EditTime({ datetime, onChange }: { datetime: number; onChange: (time: number) => void }) {
+  const [edit, setEdit] = useState<boolean>(false);
+  const [time, setTime] = useState(datetime);
+  const handleAccept = (newTime: number | null) => {
+    if (newTime) {
+      setTime(newTime);
+      onChange(new Date(newTime).getTime());
+    }
+  };
+  return (
+    <>
+      {edit ? (
+        <DateTimePicker value={time} format="MM/dd HH:mm" onAccept={handleAccept} onClose={() => setEdit(false)} />
```

Comment:
I added the package because I think the MUI Date/Time Package offers superior functionality and native support on mobile. It makes it a lot easier to implement time/date inputs.

Long-term I would like to see us migrate all of the inputs to these components to reduce complexity in the code.